datahub-web-react/src/app/homeV3/toast/notification-toast-styles.less

```diff
@@ -0,0 +1,9 @@
+.ant-notification-notice-close {
+    top: 8px;
+    right: 16px;
+}
+
+.ant-notification-notice-message,
+.ant-notification-notice-description {
+    margin-left: 36px !important;
+}
```

Comment:
hmm are these global styles necessary? is it possible to do with styled components?

not a huge deal since we only import this in `useShowToast` but unsure if it would affect anything else